cfn-resources/teams/test/README.md

```diff
@@ -35,4 +35,25 @@ The Atlas organization must be manually provided.
 
 ## Important Links
 - [API Documentation](https://www.mongodb.com/docs/atlas/reference/api-resources-spec/#tag/Teams)
-- [Resource Usage Documentation](https://www.mongodb.com/docs/atlas/access/manage-teams-in-orgs/)
\ No newline at end of file
+- [Resource Usage Documentation](https://www.mongodb.com/docs/atlas/access/manage-teams-in-orgs/)
+
+## Unit Testing Locally
```

Comment:
No need of adding this part since is in [TESTING.md](https://github.com/mongodb/mongodbatlas-cloudformation-resources/blob/master/TESTING.md.md)